api/src/pcapi/core/offerers/api.py

```diff
@@ -2095,25 +2106,30 @@ def get_venue_opening_hours_by_weekday(venue: models.Venue, weekday: models.Week
     return models.OpeningHours(weekday=weekday)
 
 
-def set_accessibility_provider_id(venue: models.Venue) -> None:
-    id_and_url_at_provider = accessibility_provider.get_id_at_accessibility_provider(
-        name=venue.name,
-        public_name=venue.publicName,
-        siret=venue.siret,
-        ban_id=venue.banId,
-        city=venue.city,
-        postal_code=venue.postalCode,
-        address=venue.address,
-    )
-    if id_and_url_at_provider:
+def set_accessibility_provider_id(
+    venue: models.Venue, id_at_provider: str | None = None, url_at_provider: str | None = None
+) -> None:
+    if not id_at_provider and url_at_provider:
```

Comment:
Es-tu certain de la condition ?
Tu demandes donc `if url_at_provider` mais n'utilises pas `url_at_provider`, tu le remplis.
Peut-être souhaitais-tu : 
```suggestion
    if not id_at_provider and not url_at_provider:
```
ou : 
```suggestion
    if not (id_at_provider or url_at_provider):
```
J'avoue ne pas cerner tout ce que ça fait d'un point de vue métier.